uint8 n_actions
uint8 i_current_action
uint8 n_steps
uint8 i_current_step
string[] ref_frames
fetch_pbd_interaction/Landmark[] objects
